frontend/app/templates/work_packages/tabs/activity.html

```diff
@@ -10,12 +10,14 @@
           <h3 class="activity-date"
               ng-if="currentDate !== (activities[$index-1].props.createdAt | date:'longDate')"
               ng-bind="currentDate"/>
-          <user-activity work-package="workPackage"
-                         activity="activity"
-                         activity-no="activityNo"
-                         is-initial="isInitial"
-                         input-element-id="'add-comment-text'">
-          </user-activity>
+            <activity-entry work-package="workPackage"
+                           activity="activity"
+                           activity-no="activityNo"
+                           is-initial="isInitial"
```

Comment:
I'll check how the current implementation handles that, otherwise we will have to work around it.